src/transaction-runner.coffee

```diff
@@ -407,20 +450,24 @@ class TransactionRunner
             else
               test.status = "fail"
 
-            gavel.validate real, transaction.expected, 'response', (validateError, result) ->
+            gavel.validate real, transaction.expected, 'response', (validateError, gavelResult) ->
               if not isValidError and validateError
                 configuration.emitter.emit 'test error', validateError, test, () ->
 
               message = ''
 
-              for resultKey, data of result
+              for resultKey, data of gavelResult
                 if resultKey isnt 'version'
                   for entityResult in data['results']
                     message += resultKey + ": " + entityResult['message'] + "\n"
 
               test.message = message
-              test.results = result
-              test.results['general'] = []
+
+              transaction.results ?= {}
```

Comment:
Because it's documented here and still not implemented. You should be able to access it in hooks for programatically bypassing some type of validation errors.